### DISCUSSION TITLE
Set inform not working http://172.30.33.2

### DISCUSSION THREAD

**Discussion by ClubspeedChris** [hassio-addons/addon-unifi#524] [General]
I have a single AP (U6 Lite) that has recently somehow got stuck on set-inform http://172.30.33.2, which I believe is an internal ip for HAOS. I ssh'd into my AP, set-inform http://192.168.96.235:8080/inform, hit enter, the AP goes offline, then back into an adopting loop.

When I run info, everytime it just defaults back to that internal 172 address. Any ideas?

![image](https://github.com/user-attachments/assets/f708838f-3efd-49fb-944c-d791544b54fd)

**Comment by SvenSowa** [comment]
Try this. Go to Unify on Hassio. Settings/System/Advanced->Inform Host. Check Override and enter the IP of your HomeAssistant server there. Then it started to work for me.
![image](https://github.com/user-attachments/assets/d88d5d01-38f5-4457-918c-da8e25c51b82)

**Comment by andtsin** [comment]
inform host field is dedicated to this site, or for all sites? if it is for all sites, its problem

**Comment by PedroBoronat** [comment]
hi, on version: Network 10.2.105, latest installed with UniFi Network Application 5.1.1 is not visible Advanced options, there is any workaround?? mi controller ip says 172.30.33.1 to all my aps and they are crashing....
<img width="1031" height="748" alt="image" src="https://github.com/user-attachments/assets/0ca0ce29-db6d-4374-9c51-25d50530d54e" />
